### PR TITLE
Convert enum to enum34

### PR DESCRIPTION
**Pull request by wallacbe**

### CODE REVIEW

File: openhtf/exe/test_state.py [hunk 1]
```diff
@@ -50,10 +50,16 @@ class TestState(object):
     config: The config being used for this test.
     test: openhtf.Test instance describing the test to run.
   """
-  State = Enum(
-      'RUNNING', 'ERROR', 'TIMEOUT', 'ABORTED', 'WAITING', 'FAIL', 'PASS',
-      'CREATED'
-  )
+  
+  class State(Enum):
```
Comment (fahhem) [inline openhtf/exe/test_state.py]: This could still be in the same style:

```py
State = Enum('State', ['RUNNING', ...])
```

**Comment by fahhem** [comment]
You might want to run 'git rebase master' to remove the unnecessary merge

**Comment by wallacbe** [comment]
Made the suggested enum format changes.

I also rebased at the commit before that merge then rebased master then re-pushed

**Comment by fahhem** [comment]
Odd, the merge commit should have gone away.

**Pull request closed without merging**